app/src/main/res/layout/dialog_results.xml

```diff
@@ -343,6 +343,7 @@
 
         <include layout="@layout/expanded_results_box"
             android:id="@+id/expandedResultsBox"
+            android:visibility="gone"
```

Comment:
Unicode will turn into their emoji counterparts on most devices. And it isn't pleasant looking at one Unicode symbol and one emoji.